### ISSUE TITLE
Use social media metrics?

### ISSUE THREAD

**Issue by agnelvishal** [yacy/yacy_search_server#288]
Are there any plans to use Facebook likes, Reddit upvotes etc as a metric for ranking the webpages? The Reddit api is at https://www.reddit.com/dev/api/ and facebook api is at https://developers.facebook.com/

**Comment by frankenstein91** [comment]
It would be a good idea, but against some basic ideas of data privacy implementations in Yacy. For example, no search words are transferred between the Yacy peers, only hash parts.

**Comment by agnelvishal** [comment]
Yes. I also thought if it would be a privacy concern.But people upvote/like/share a webpage to let others know that they recommend it. They do it to for the public. So privacy shouldn't be a concern here I think.
Also we would only get the number of upvotes from the api. Not who upvoted
